docs/CONTRIBUTING.md

```diff
@@ -45,15 +46,90 @@ Some helpful guides:
 1. Run `npm start` to start telescope.
    _If you get a series of errors, you may have to start redis-server depending on your installation configuration, do this by running the command `redis-server` in a seperate command window)._
 
-#### Logging
+## Workflow in git and GitHub
+
+When working on fixing bugs, please use the following workflow:
+
+1. If you haven't done so already, add an `upstream` remote so you can stay in sync:
+   ```
+   git remote add upstream https://github.com/Seneca-CDOT/telescope.git
+   ```
+1. Before you do any new work, always update your `master` branch:
+   ```
+   git checkout master
+   git pull upstream master
+   ```
+1. Create a branch for your work, using the issue number:
+   ```
+   git checkout -b issue-123
+   ```
+1. Do your work, add your files manually, and commit early, commit often!
+   ```
+   ...edit files
+   git add file1 file2 ...
+   git commit -m "Updating file1 and file2 to do ..."
+   ```
+1. Run the test suite, using `npm test`. Fix any lint errors, warnings, or other failures (NOTE: if you're not sure what an eslint rule means, [look it up in the docs](https://eslint.org/docs/rules/)):
+   ```
+   npm test
+   ...if there are lint errors, try having eslint fix them for you
+   npm run eslint-fix
+   npm test
+   ...manually fix any errors yourself, rerunning npm test each time to confirm
+   ```
+1. When you're done, push to your fork, using the same name as your branch:
+   ```
+   git push origin issue-123
+   ```
+1. This will give you a link to GitHub to create a Pull Request.
+1. Create your Pull Request, and give it a title `Fix #123: short description of fix` (NOTE: the "Fix #123" part, which will [automatically close the issue](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords) when this gets merged) as well as a full description to talk about everything you did, any follow-up work that's necessary, how to test the code, etc.
```

Comment:
GitHub says [here](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords):
> "Closes #123" or "Closes: #123" in your pull request **description** or **commit message**.

I'm not sure if it applies to titles as well.

Reply:
Try it out with a test issue, test branch and test PR!! Lettuce know what happens

Reply:
Works in either place ime.